.circleci/config.yml

```diff
@@ -239,459 +333,146 @@ jobs:
+            - installtorchcpu
+          cachename: ut36
+          marker: unit
 
-  unittests_38:
-    <<: *standard_cpu38
+  unittests_37:
+    executor: standard_cpu38
     working_directory: ~/ParlAI
     parallelism: 2
     steps:
-      - checkout
-      - <<: *fixgit
-      - restore_cache:
-          key: deps-20200921-ut38-{{ checksum "requirements.txt" }}
-      - <<: *setup
-      - <<: *installtorchcpu
-      - <<: *installdeps
-      - save_cache:
-          key: deps-20200921-ut38-{{ checksum "requirements.txt" }}
-          paths:
-            - "~/venv/bin"
-            - "~/venv/lib"
-      - <<: *findtests
-      - restore_cache:
-          key: data-20200904-ut38-{{ checksum "teststorun.txt" }}
-      - run:
-          name: Unit tests (py38)
-          no_output_timeout: 60m
-          command: coverage run -m pytest --junitxml=test-results/junit.xml -m unit -v
-      - <<: *codecov
-      - save_cache:
-          key: data-20200904-ut38-{{ checksum "teststorun.txt" }}
-          paths:
-            - "~/ParlAI/data"
-      - store_test_results:
-          path: test-results
+      - runtests:
+          more_installs:
+            - installtorchcpu
+          cachename: ut37
+          marker: unit
 
-  unittests_37:
-    <<: *standard_cpu37
+  unittests_38:
+    executor: standard_cpu38
     working_directory: ~/ParlAI
-    parallelism: 8
+    parallelism: 2
     steps:
-      - checkout
-      - <<: *fixgit
-      - restore_cache:
-          key: deps-20200921-ut37-{{ checksum "requirements.txt" }}
-      - <<: *setup
-      - <<: *installtorchcpu
-      - <<: *installdeps
-      - save_cache:
-          key: deps-20200921-ut37-{{ checksum "requirements.txt" }}
-          paths:
-            - "~/venv/bin"
-            - "~/venv/lib"
-      - <<: *findtests
-      - restore_cache:
-          key: data-20200904-ut37-{{ checksum "teststorun.txt" }}
-      - run:
-          name: Unit tests (py37)
-          no_output_timeout: 60m
-          command: coverage run -m pytest --junitxml=test-results/junit.xml -m unit -v
-      - <<: *codecov
-      - save_cache:
-          key: data-20200904-ut37-{{ checksum "teststorun.txt" }}
-          paths:
-            - "~/ParlAI/data"
-      - store_test_results:
-          path: test-results
+      - runtests:
+          more_installs:
+            - installtorchcpu
+          cachename: ut38
+          marker: unit
 
   unittests_gpu14:
-    <<: *gpu
+    executor: gpu
     working_directory: ~/ParlAI
     parallelism: 2
     steps:
-      - checkout
-      - <<: *fixgit
-      - restore_cache:
-          key: nvidia-downloads-20200516
-      - <<: *setupcuda
-      - save_cache:
-          key: nvidia-downloads-20200516
-          paths:
-            - "~/nvidia-downloads/"
-      - restore_cache:
-          key: deps-20200921-gpu14-{{ checksum "requirements.txt" }}
-      - <<: *setup
-      - <<: *installtorchgpu14
-      - <<: *installdeps
-      - <<: *installdetectrondeps
```

Comment:
Hmm how do things like `installdetectrondeps` get run now if they're not explicitly called?

Reply:
This was missed, it needs to be added back in